im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,426 @@
+        <else-if type="book" match="any">
+          <group>
+            <text macro="author"/>
+            <text variable="title" font-style="italic" suffix="."/>
+            <text macro="band-aus-reihe"/>
+            <text variable="publisher-place" prefix=" " suffix=": "/>
+            <text variable="publisher"/>
+            <text macro="edition-number"/>
+            <text macro="year-date"/>
+            <text macro="pages"/>
+          </group>
+        </else-if>
+        <else-if type="webpage" match="any">
+          <group>
+            <text macro="author"/>
+            <text variable="title" suffix=". "/>
+            <text variable="container-title" font-style="italic" prefix="online: " suffix=", "/>
+            <text macro="date"/>
+            <text macro="access"/>
+          </group>
+        </else-if>
+        <else-if type="post-weblog" match="any">
+          <group delimiter=",">
+            <group prefix=" ">
+              <text macro="author" suffix=" "/>
+              <text variable="title" suffix=". "/>
+              <text term="in" text-case="lowercase" suffix=": "/>
+              <text variable="container-title" font-style="italic" suffix=", "/>
+              <text macro="date"/>
+              <text macro="access"/>
+            </group>
+          </group>
+        </else-if>
+        <else-if type="article-journal article article-magazine article-newspaper" match="any">
+          <group>
+            <text macro="author" suffix=" "/>
+            <text variable="title" suffix="."/>
+            <text variable="container-title" font-style="italic" prefix=" In: " suffix=". "/>
+            <text variable="volume" suffix=", "/>
+            <text variable="issue"/>
+            <text macro="year-date"/>
+            <text variable="publisher-place" suffix=": "/>
+            <text macro="publisher"/>
+            <text macro="pages"/>
+          </group>
+        </else-if>
+        <else-if type="thesis" match="any">
+          <group>
+            <text macro="author"/>
+            <text variable="title" font-style="italic" suffix=". "/>
+            <text variable="genre" suffix=", "/>
+            <text variable="publisher-place" suffix=": "/>
+            <text variable="publisher"/>
+            <text macro="year-date"/>
+          </group>
+        </else-if>
+        <else-if type="paper-conference article-newspaper article-magazine article-journal" match="any">
+          <group>
+            <text macro="author"/>
+            <text variable="title" font-style="normal" suffix=". "/>
+            <text variable="container-title" font-style="italic" prefix="in: "/>
+            <text macro="year-date"/>
+            <text macro="pages" prefix=", "/>
+          </group>
+        </else-if>
+        <else-if type="motion_picture" match="any">
+          <group>
+            <text variable="title" font-style="italic"/>
+            <text variable="original-title" prefix=" (" suffix=". "/>
```

Comment:
If you want to place parentheses around a group of items, set them as affixes on a group. The way you have this now, if you have a film w/o director or original titles, you'll have unbalanced parentheses.